packages/expo-cli/src/commands/init.ts

```diff
@@ -415,7 +415,8 @@ async function promptForBareConfig(
   } else {
     ({ projectName } = await prompt({
       name: 'projectName',
-      message: 'What is the name of your project?',
+      message: 'What is your app called?',
```

Comment:
not crazy about this wording change

`What would you like to name your app?` maybe seems better?